spec/index.bs

```diff
@@ -13389,6 +13431,9 @@ dictionary GPURenderBundleEncoderDescriptor
          : GPURenderPassLayout {
     boolean depthReadOnly = false;
     boolean stencilReadOnly = false;
+
+    // Requires "interited-bundle-bind-groups" feature.
+    sequence<GPURenderBundleInheritedBindGroupEntry> inheritedBindGroups = [];
```

Comment:
Alternatively we could match `GPUPipelineLayoutDescriptor` with `sequece<GPUBindGroupLayout?>`. This would force that the inherited bindgroups are a prefix of all the groups, that is you can inherit groups 0 , 1, 2 but not just 0, 2. This would be a useful way to encode in the API that the groups should be ordered from less frequently changed to more frequently changed.